packages/terra-dialog-modal/src/DialogModal.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
```

Comment:
If it's absolutely necessary to disable this, disable it for the specific line only.

Reply:
Changed 126317f